lib/iframe-browser.js

```diff
@@ -83,6 +83,10 @@ define(function (require, exports, module) {
             var iframe = getBrowserIframe();
             if(iframe) {
                 iframe.src = url;
+                var detachedWindow = getDetachedPreview();
```

Comment:
Flatten this a bit:

```js
 if(!url) {
  return;
}

var iframe = getBrowserIframe();
if(iframe) {
  iframe.src = url;
}

var detachedWindow = getDetachedPreview();
if(detachedWindow) {
  detachedWindow.location.replace(url);
}
```

Also, update the comment above this function to reflect that it's more than just the iframe.